frozendict/FrozenDict.py

```diff
@@ -43,9 +45,25 @@ def __new__(cls, *args, **kwargs):
     def __init__(self,
                  value: Mapping[KT, VT_co] = None,
                  *,
-                 homogeneous_type=False,
-                 remove_none_value=False,
+                 homogeneous_type: bool = False,
+                 remove_none_value: bool = False,
+                 no_copy: bool = False,
```

Comment:
I would much appreciate to be able to set a "global" (static) flag (once) to have 'True' as default for the 'no_copy' parameter.

Reply:
isn't that very dangerous if the flag is set somewhere but is not considerate in another part of the code ? 
PS: I might not be too familiar with python scoping but it seems that if 2 dev work on different part of the code with one using that option could lead to terrible problems.

Reply:
Indeed, it would be confusing to the user if the docs & IDE completion specify the default to be `False` and it's been overridden to be `True` somewhere else. Is there a Python feature/pattern that could solve this, like overriding `__init__` at import-time/at the file level?

Reply:
As discussed, I agree that a global flag would not be desirable, and we should rather provide a `NoCopyFrozenDict` "variant".